{"levelname": "ERROR", "asctime": "2024-04-08T18:30:08.008Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 79, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "83179a5e-2995-4ef2-9a27-1fda9b741487", "collection_version_id": "f1012cb5-b6b2-4d30-a5d2-51468539252e", "errors": ["error message"]}
